### PR TITLE
feat: Add dev deploy yaml script

### PR DESCRIPTION
**Pull request by uchukwu**
Update the GitHub Action to deploy api-dev to ECS

### CODE REVIEW

**Comment by losvedir** [comment]
You can also delete the existing `.github/workflows/dev.yml` workflow.

**Comment by uchukwu** [comment]
> You can also delete the existing `.github/workflows/dev.yml` workflow.

Completed the removal of `dev.yml`

File: .github/workflows/deploy-dev.yml [hunk 1]
```diff
@@ -1,9 +1,6 @@
-name: Deploy to Dev
+name: Deploy to Dev (ECS)
 
-on:
-  workflow_dispatch:
-  push:
-    branches: [master]
```
Comment (losvedir) [inline .github/workflows/deploy-dev.yml:6]: Ah, the

```
push:
  branches: [master]
```

shouldn't have been removed. That's the workflow trigger which causes api-dev to be automatically deployed to when we merge a PR, which is behavior we like in our dev environments.

Comment (uchukwu) [reply .github/workflows/deploy-dev.yml:6]: Completed the change 👍

**Review by losvedir — APPROVED** [review]
👍

**Pull request merged**